html/apps/applist/style.css

```diff
@@ -0,0 +1,58 @@
+/* Mimicking a product grid, don't copy that in your theme! */
+.content { width: 80%; margin: 20px auto; }
+.span2 { width: 32%; padding: 0; margin: 0; float: left; margin-right:1%; }
+.span2 img { max-width: 100%; }
+p { margin: 0.5em 0; }
+.reveal p { margin: 0; }
+body { font-family: Lato, sans-serif; }
+.span2 p { text-align: center; }
+
+/* ===============================================
+// Reveal module
+// =============================================== */
+
+.reveal .hidden { display: block !important; visibility: visible !important; }
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use of !important](https://www.codacy.com/app/sudheesh1995/loklak_server/file/2611673807/issues/source?bid=3288751&fileBranchId=3310993#l14)